Clinical-Genomics/scout
# Add clinVar CNVs track in IGV

fix #1757 and fix #1793

This works, but it's using the igv.js library that represents the patched version immediately before a new release (I don't know when the new release will happen). I suggest we merge and when they release the new version we update our code (I'm working on another PR involving the igv viewer anyway).  

This is what both clinvar tracks look like after the fix:
![image](https://user-images.githubusercontent.com/28093618/77636843-01200380-6f55-11ea-91be-4e9cbf4077e9.png)


**How to test**:
1. Install this branch on scout-stage in clinical-db
1. Check some variants in IGV from a case which has alignments available (example: https://scout-stage.scilifelab.se/cust000/643594-300M) and make sure that clinVar CNV track is present
1. Go to the IGV browser of the case above and check this region: chr21:47,408,993-47,409,093. Bug #1793 should be fixed.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
